std/range/package.d

```diff
@@ -7250,7 +7250,7 @@ if (isInputRange!Range)
     }
 
     // Assertion only.
-    private void dbgVerifySorted()
+    private auto dbgVerifySorted()
```

Comment:
Why does this work? I copied the code and the deduced return type remained `void`.